### PR TITLE
Address DR-20

### PR DESCRIPTION
**Pull request by CanOfBees**
**JIRA Ticket**: [DR-20](https://jira.lib.utk.edu/projects/DR/issues/DR-20)

# What does this Pull Request do?
Adds a sample MODS DSID and a roth_post_process.xsl for removing empty elements and empty attributes.

# What's new?
A transform that removes empty elements and empty attributes.

# How should this be tested?

Please use Option 1
1. Option 1:
	* Import the Form
	* Associate it with a content model
	* Apply any additional / related transforms
	* Create a new record and select the newly associated form
	* Edit that record to see if the form still behaves correctly
2. Option 2:
	* vagrant destroy
	* vagrant up

# Interested parties
@markpbaggett

### CODE REVIEW

**Comment by markpbaggett** [comment]
@CanOfBees, review is underway.

**Comment by markpbaggett** [comment]
Also, we should really change the p.r. template here for testing. You can't test this with "vagrant destroy" / "vagrant up."  

Since I just posted that comment to this p.r., do you mind just adding this to this pull request.  That way, no need to open another.

**Comment by CanOfBees** [comment]
@markpbaggett agreed. i'll add the change.

**Comment by markpbaggett** [comment]
Okay, how about something like this:

```
## How should this be tested

Describe what steps to take to test this change.

**Examples:**
* Import the Form, Associate it with a content model, Apply any additional / related transforms, Create a new record and select the newly associated form, Edit that record to see if the form still behaves correctly
* vagrant destroy and vagrant up
* Associate transform to existing xml form

```
Not necessarily like that, but you know what I mean.

**Comment by CanOfBees** [comment]
@markpbaggett take a look at the new verbiage. Thanks!

**Review by markpbaggett — CHANGES_REQUESTED** [review]
line 30 of PULL_REQUEST_TEMPLATE is wrong.  

Should be:  * `sudo cp $REPO_NAME/dir-path/transform.xsl /var/www/drupal/sites/all/modules/islandora_xml_forms/builder/self_transforms/`

Everything else looks good though. Works!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -14,15 +14,24 @@ Example:
...
+    * Import the Form
+    * Associate it with a content model
+    * Apply any additional related transforms
+    * Create a new record and select the newly associated form
+    * Edit that record to verify proper CRUD behavior
+* `vagrant destroy -f && vagrant up`
+* Associating a post-processing transform
+    * `vagrant destroy -f && vagrant up && vagrant ssh`
+    * `git clone https://github.com/$USER_NAME/$REPO_NAME`
+    * `sudo cp $REPO_NAME/dir-path/transform.xsl /var/www/drupal/sites/all/modules/islandora_xml_transforms/builder/self_transforms/`
```
Comment (markpbaggett) [inline .github/PULL_REQUEST_TEMPLATE.md]: This should be: 

* `sudo cp $REPO_NAME/dir-path/transform.xsl /var/www/drupal/sites/all/modules/islandora_xml_forms/builder/self_transforms/`

**Review by markpbaggett — CHANGES_REQUESTED** [review]
One little change.  All else looks good though.

**Comment by CanOfBees** [comment]
@markpbaggett thanks for catching that.

**Review by markpbaggett — APPROVED** [review]

**Comment by markpbaggett** [comment]
@CanOfBees, I've merged.  Feel free to mark DR-20 as "Done."  I created a DR-21.  Either of us can do that.

**Pull request merged**